src/System.Composition.Convention/src/System.Composition.Convention.csproj

```diff
@@ -61,5 +61,21 @@
       <Name>System.Composition.AttributedModel</Name>
     </ProjectReference>
```

Comment:
Hm, I'm not sure. These System.Composition ones are a bit special and I think I'd rather just keep them as-is. They sort of act as one "unit" anyways since they have historically been a single package.